.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
```

Comment:
check please is it required to specify Java version?
We agreed to use J11, think it can be configured also because it's very important for build.